tasks/schema.js

```diff
@@ -272,7 +272,20 @@ async function migrateRelations(context) {
     )
     .flat();
 
-  await apiV9.post("/relations", [...relationsV9, ...systemFields]);
+  const mappedRelations = [...relationsV9, ...systemFields].map( realtion => {
+    return {
+      meta: realtion,
+      field: realtion.many_field,
+      collection: realtion.many_collection,
+      schema: {
+          table: realtion.many_collection,
+          column:  realtion.many_field,
+          foreign_key_table: realtion.one_collection,
+          foreign_key_column: realtion.id,
+      },
+    }
+  } );
```

Comment:
Instead of re-mapping it here, lets make sure we construct the right shape above (here: https://github.com/directus-community/migration-tool/pull/32/files#diff-1f70c04c18a332eefd09d4aa19d19b54bb9fe732bdaa2f71d033e61e82be0aeeR247-R273)

Reply:
I refactored the shape